components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/ApplicationAuthenticatorService.java

```diff
@@ -68,6 +112,28 @@ public LocalAuthenticatorConfig getLocalAuthenticatorByName(String name) {
         return null;
     }
 
+    /**
+     * Retrieve both USER and SYSTEM defined Local Application Authenticator configuration by name.
+     *
+     * @param name                  The name of the Local Application Authenticator configuration.
+     * @param tenantDomain          Tenant domain.
+     *
+     * @return Retrieved LocalAuthenticatorConfig.
+     * @throws AuthenticatorMgtException If an error occurs while retrieving the authenticator configuration by name.
+     */
+    public LocalAuthenticatorConfig getLocalAuthenticatorByName(String name, String tenantDomain)
+            throws AuthenticatorMgtException {
+
+        /* First, heck whether an authenticator by the given name is in the system defined authenticators list.
```

Comment:
there's a typo in the comment 
heck -> check

Reply:
Addressed.